Automattic/woocommerce-payments
# Use type wcpay_sepa for payment token to avoid conflicts with stripe plugin

closes #1605

#### Changes proposed in this Pull Request

Change the type name of the sepa payment token to `wcpay_sepa` this avoids the conflicts that can happen when the Stripe plugin is installed at the same time as WooCommerce Payments. The class name is also changed to match.

Only downside to doing this is existing sepa payment tokens will essentially stop working. Should not be a problem for anyone since it is hidden behind a feature flag. 

#### Testing instructions
* Checkout with SEPA. Check save payment method.
* Try again with saved method
* It should work

-------------------
